src/components/CurrentWeather.jsx

```diff
@@ -1,12 +1,21 @@
 import PropTypes from 'prop-types';
 
-const CurrentWeather = ({ currentSearch, currentTemp }) => (
-  <p>{`The current weather in ${currentSearch} is ${currentTemp} C.`}</p>
-);
+import useAPIRequest from './CustomHooks/useAPIRequest';
+import { REQUEST_TYPES } from '../common/constants';
+
+const CurrentWeather = ({ cityId, cityName }) => {
+  const { current } = useAPIRequest(REQUEST_TYPES.current, cityId);
+
+  return current ? (
+    <p>{`The current weather in ${cityName} is ${current.temperature} C.`}</p>
```

Comment:
Just a hint, maybe overcomplicating but still... you may also handle the situation when for some reason temperature property is not present on current object. What will user see then? 

Plus, the weather may be fine or bad or cloudy, or foggy...but the weather is 10 degrees? I think the wording may be a bit different.